test/e2e/storage/vsphere/persistent_volumes-vsphere.go

```diff
@@ -67,11 +67,7 @@ var _ = utils.SIGDescribe("PersistentVolumes:vsphere", func() {
 		clientPod = nil
 		pvc = nil
 		pv = nil
-		nodes := framework.GetReadySchedulableNodesOrDie(c)
-		if len(nodes.Items) < 1 {
-			framework.Skipf("Requires at least %d node", 1)
-		}
-		nodeInfo = TestContext.NodeMapper.GetNodeInfo(nodes.Items[0].Name)
+		nodeInfo = GetReadySchedulableRandomNodeInfo()
```

Comment:
nit: The original code made skip if the node number is 0, and the changed code makes failure with `gomega.Expect(nodesInfo).NotTo(gomega.BeEmpty())` in GetReadySchedulableRandomNodeInfo().
I guess the new behavior is not bad because that is the same as the other test code.
This is just a comment if somebody face this new behavior.